packages/modules/itwin-viewer-react/src/components/Viewer.tsx

```diff
@@ -50,7 +51,7 @@ export const Viewer: React.FC<ViewerProps> = ({
     if (!iModelJsInitialized) {
       const authClient = getAuthClient(authConfig);
       Initializer.initialize(
-        { authorizationClient: authClient },
+        { authorizationClient: authClient, toolAdmin },
```

Comment:
same change needed in packages\modules\itwin-viewer-react\src\components\BlankViewer.tsx